web/src/project/ProjectSummary.tsx

```diff
@@ -310,8 +319,39 @@ export const ProjectSummary = () => {
     const titleCase = (s: string) => {
         return s[0].toUpperCase() + s.slice(1).toLowerCase();
     };
+    console.log(summary?.sequence_stats);
```

Comment:
Should the batch table only show if there are other keys than "no-batch" - seems sort of irrelevant for like perth-neuro or something.

Reply:
Can do - would it make more sense to return an empty object from the backend if there are no other keys or should we check on the front end for other keys?

Reply:
Yeah I think empty object if there are no other keys, make it easy for you. Total sequences are available in other places.

Reply:
Returns {} if 'no-batch' is the only key